de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Server_Manual.adoc

```diff
@@ -0,0 +1,236 @@
+= Virtual Satellite 4 Core: Server Manual
+:imagesdir: images
+:title-logo-image: images/title/VirtualSatellite_Server.jpg
+:toc:
+:toclevels: 3
+:experimental: 
+
+== Introduction
+
+Virtual Satellite can be run headless as a server that allows you to connect Virtual Satellite to a network, e.g. the Internet.
+The REST API can be used to access and manipulate the Virtual Satellite model remotely and without the desktop interface.
+It provides the Model in form of Resources that can be manipulated via the standard HTTP commands GET, PUT, POST and DELETE.
+
+The Server functions as a general-purpose interface to connect Virtual Satellite to your own application.
+For example the API could be called via JavaScript, that runs inside your Browser, from within a Python program or directly from the command line.
+
+You will learn about the Servers Architecture in the next chapter.
+If you are interested in connecting the Server to your own application, refer to <<The REST API>>.
+Or if you want to set up your own Server, then refer to <<How to run your own Headless Virtual Satellite Server>>.
+
+== Architecture of the Virtual Satellite Server
+
+In this chapter an overview and a more in depth description of the Servers Functionality and Components is given.
+
+=== Overview
+
+The Figure <<RestServerOverview>> gives an overview of the Server and its environment.
+
+.Overview of the Headless REST Server in the Virtual Satellite environment
+[#RestServerOverview]
+image::chapterServerOverview/REST_server_overview.png[Server Overview]
+
+A Server instance can maintain multiple _Server Repositories_, with each repository connected to a _git_ or _SVN repository_.
+_REST Clients_ can access the Server using the HTTP commands GET, PUT, DELETE and POST. 
+So a Client can read or write into the Virtual Satellite model. 
+The server provides functionality for administrators, that can manipulate the server configuration, and users.
+For that purpose it uses authentication and authorization.
+// For authentication purposes it is planned to an LDAP server can be connected to the REST server.
+// The connection can be secured via HTTPS.
+Every user can only access certain provided Server Repositories.
+
+The connected SVN/git repositories can also be directly changed from _Clients using the Virtual Satellite UI_.
```

Comment:
```suggestion
The connected SVN/git repositories can also be manipulated from _Clients using the Virtual Satellite UI_.
```
change sounded for me like you could change the remote or other git/svn properties; directly sounded like you could use UI to direktly manipulate the checked-out repo within the server...

Reply:
Yeah I will change it